semgrep/semgrep/test.py

```diff
@@ -142,11 +160,18 @@ def join_keys(a: Dict[str, Any], b: Dict[str, Any]) -> Set[str]:
         for check_id in join_keys(comment_lines[file_path], reported_lines[file_path]):
             all_reported = set(reported_lines[file_path][check_id])
             expected = set(comment_lines[file_path][check_id])
+            oked = set(ok_lines[file_path][check_id])
             ignored = set(ignore_lines[file_path])
 
+            reported_oked_lines = oked.intersection(all_reported)
+            if reported_oked_lines:
```

Comment:
@minusworld here's the error if an ok'ed line is reported. It'd be a good idea to go over our existing tests and mark the ok'ed lines with their specific rule-id. That way we'll be notified if we have any false positives.